### PR TITLE
[6.4.x] RHBPMS-4184,JBPM-5277 - getTasksAssignedAsPotentialOwner doesn't allow groups searching, only actorid

### PR DESCRIPTION
**Pull request by mrietveld**
PR Set: 
- https://github.com/droolsjbpm/droolsjbpm-knowledge/pull/160
- https://github.com/droolsjbpm/jbpm/pull/560
- https://github.com/droolsjbpm/droolsjbpm-integration/pull/605

### CODE REVIEW

File: kie-remote/kie-remote-client/src/main/java/org/kie/services/client/api/command/TaskServiceClientCommandObject.java [hunk 1]
```diff
@@ -682,6 +683,13 @@ public Task getTaskById( long taskId ) {
     }
 
     @Override
+    public List<TaskSummary> getTasksByGroup(List<String> groupIds) {
+        GetTasksByGroupCommand cmd = new GetTasksByGroupCommand();
+        cmd.getGroupIds().addAll(groupIds);
+        return executeCommand(cmd);
```
Comment (psiroky) [inline kie-remote/kie-remote-client/src/main/java/org/kie/services/client/api/command/TaskServiceClientCommandObject.java]: I am not sure about this, but will ask just to double-check: do we need to handle the new command class in e.g. https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.4.x/kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/jaxb/JaxbCommandsResponse.java ? Similarly as the `GetTaskByWorkItemIdCommand` which was fixed there recently?

**Comment by mrietveld** [comment]
Jenkins please retest

File: kie-remote/kie-remote-client/src/main/java/org/kie/services/client/api/command/TaskServiceClientCommandObject.java (hunk 1 above)
Comment (mrietveld) [inline kie-remote/kie-remote-client/src/main/java/org/kie/services/client/api/command/TaskServiceClientCommandObject.java]: Nice catch, @psiroky! 

Fortunately, it's taken care of here: 
https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.4.x/kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/jaxb/JaxbCommandsResponse.java#L185

**Comment by psiroky** [comment]
+1

**Pull request merged**